proxy/config_test.go

```diff
@@ -94,6 +94,32 @@ func TestWhenHasErgoFile(t *testing.T) {
 
 	})
 
+	t.Run("It removes a service", func(tt *testing.T) {
+		fileContent, err := ioutil.ReadFile("../ergo")
+
+		if err != nil {
+			tt.Skipf("Could not load initial .ergo file")
+		}
+
+		//we clean after the test. Otherwise the next test will fail
+		defer ioutil.WriteFile("../.ergo", fileContent, 0755)
+
+		service := NewService("testservice", "http://localhost:8080")
+
+		if err := RemoveService("../.ergo", service); err != nil {
+			tt.Errorf("Expected no error while removing service. Got %v\n", err)
+		}
+
+		newFileContent, err := ioutil.ReadFile("../ergo")
+		if err != nil {
+			tt.Skip("Could not load initial .ergo file")
+		}
+
+		if !(len(fileContent) > len(newFileContent)) {
```

Comment:
Could we ensure that we have a service (lets say "servicetoberemoved") at the beginning of the test and at the end ensure it is not there anymore?
Would be nice to ensure that the others services won't be affected by the command as well :)

Reply:
I've added this servicetoberemoved onto the .ergo file

Reply:
👍